batch_notification_processor/lambda_function.py

```diff
@@ -7,7 +7,7 @@
 import boto3
 
 from batch_processor import BatchProcessor
-from communication_management import CommunicationManagement
+from comms import CommunicationManagement
```

Comment:
It looks like we have renamed this module from `communication_management` to `comms`, none of the commit messages explain why, was this intentional? Is this an ambiguous module import problem?